components/maincontroller.js

```diff
@@ -18,6 +18,12 @@
     var init = function () {
 
         resetPlaybackScope($scope);
+<<<<<<< HEAD
```

Comment:
This looks wrong ;)

Reply:
That's embarrassing 😅. I must been a bit too fast on the trigger. Will fix tonight

Reply:
After taking a closer look I could've sworn I fixed these 🤔